tests-beta/android/MASVS-RESILIENCE/MASTG-TEST-0x41.md

```diff
@@ -0,0 +1,26 @@
+---
+platform: android
+title: Testing if StrictMode is enabled
+id: MASTG-TEST-0X41
+apis: [StrictMode]
+type: [dynamic]
+weakness: MASWE-0094
+---
+
+## Overview
+
+This test verifies if an app enables [StrictMode](../../../0x05i-Testing-Code-Quality-and-Build-Settings/#strictmode) in production.
```

Comment:
```suggestion
This test checks whether an app enables [`StrictMode`](../../../0x05i-Testing-Code-Quality-and-Build-Settings/#strictmode) in production. While useful for developers to log policy violations such as disk I/O or network operations in production apps, leaving `StrictMode` enabled can expose sensitive implementation details in the logs that could be exploited by attackers.
```

Reply:
Shouldn't the second part of the paragraph be better in the weakness? I thought once you are in the test, you should already have understood the problem?